packages/api/src/controllers/stream.ts

```diff
@@ -897,18 +897,17 @@ app.post(
     // compatibility with /setactive recordging/webhooks handling)
     const issues =
       payload.is_active && !payload.is_healthy
-        ? payload.human_issues ||
-          (payload.issues ? [payload.issues] : undefined)
-        : undefined;
+        ? payload.human_issues || (payload.issues ? [payload.issues] : null)
+        : null;
```

Comment:
In case of an array, we have a reasonable default value that does indicate "no issues". This would be the empty array `[]`. WDYT of using that instead of `null` here?

Reply:
Mmm yeah makes sense! The tests where checking for `toBeUndefined` tho, will change them to look for an empty array, sounds good?

Reply:
Yeah that made sense! Keeping as null sounds ok anyway :)